.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  publish-github-release:
+    runs-on: ubuntu-latest
+    steps:
+      # Publishes new Release as new git tag (vX.Y.Z) is pushed to repository
+      - uses: release-drafter/release-drafter@v5
+        with:
+          config-name: release-drafter-config.yml
```

Comment:
Pls rename this config file accordingly as well

Reply:
This config-name relates to the tool we are using: https://github.com/marketplace/actions/release-drafter (so we use release-drafter-config.yml to configure release-drafter :p ). But I will change the name.